sdk/compatibility/bazel_tools/testing.bzl

```diff
@@ -759,6 +759,18 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        "end": "2.10.0-snapshot.20240821.12933.0",
```

Comment:
Because the ledger-api-test-tool from sdk <= 2.9.x expects the trace to be of the form: 
```
 "Interpretation error: Error: (User abort: Assertion failed.?|Unhandled (Daml )?exception: [0-9a-zA-Z\\.:]*@[0-9a-f]*\\{ message = \"Assertion failed\" \\}\\. [Dd]etails(: |=)Last location: \\[[^\\]]*\\], partial transaction: root node)"
```